tests/unit/multi_table/test_hma.py

```diff
@@ -277,20 +277,20 @@ def test__finalize(self):
 
         sampled_data = {
             'users': pd.DataFrame({
-                'user_id': [0, 1, 2],
-                'name': ['John', 'Doe', 'Johanna'],
-                'additional_column': [0.1, 0.2, 0.3],
-                'another_additional_column': [0.1, 0.2, 0.5]
-            }, dtype=np.int64),
```

Comment:
Could we not just remove this? @pvk-developer I'm not sure what this `dtype=np.int64` is for.

Reply:
Yes, I was not sure too. In the test documentation, it's written:
`The process consists of applying the proper data types to each table` so I thought the dtype was important for the foreign keys at least but maybe we can just remove it.